src/Controller/WopiController.php

```diff
@@ -88,9 +89,14 @@ protected function wopiCheckFileInfo(FileInterface $file, UserInterface $user, b
       'SupportsRename' => FALSE,
     ];
 
-    $user_picture = $user->user_picture?->entity;
-    if ($user_picture) {
-      $response_data['UserExtraInfo']['avatar'] = $this->fileUrlGenerator->generateAbsoluteString($user_picture->getFileUri());
+    if ($user->hasField('user_picture')) {
+      $user_picture = $user->get('user_picture')->entity;
+      if (
+        $user_picture instanceof FileInterface &&
+        $user_picture->getFileUri() !== NULL
+      ) {
```

Comment:
We could have just one big if instead of two levels, if we do assignment in the condition.\
Some people like that, others don't.
I currently prefer assignment to be separate from the condition, but my opinion on that may change again.